src/ddo/MetaData.ts

```diff
@@ -194,6 +194,8 @@ export interface MetaDataMain {
     ercType?: ERCType
 
     nftType?: NeverminedNFTType
+
+    isDTP?: boolean
```

Comment:
is this used anywhere?

Reply:
Yes for example here:
https://github.com/nevermined-io/cli/blob/main/src/commands/assets/registerAsset.ts#L83